src/main/java/org/medicmobile/webapp/mobile/RequestStoragePermissionActivity.java

```diff
@@ -74,6 +74,13 @@ public class RequestStoragePermissionActivity extends FragmentActivity {
 	public void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+			trace(this, "RequestStoragePermissionActivity :: READ_EXTERNAL_STORAGE permission is ignored in Android 13+.");
+			setResult(RESULT_OK, createResponseIntent());
+			finish();
```

Comment:
Just in case if someone is wondering

> You can call finish() from within this function, in which case onDestroy() will be immediately called after onCreate(Bundle)

[source](https://developer.android.com/reference/android/app/Activity#onCreate(android.os.Bundle))